deploy/docker/cp-run-policy-manager/policy_manager.py

```diff
@@ -0,0 +1,188 @@
+MONITORING_PERIOD_SEC = int(os.getenv('RUN_OWNER_POLICY_POLL_PERIOD_SEC', 5))
+
+
+def log_message(message):
+    print('[{}] {}'.format(datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S'), message))
+
+
+def get_custom_resource_api():
+    config.load_kube_config()
+    return client.CustomObjectsApi()
+
+
+def load_all_active_policies():
+    api = get_custom_resource_api()
+    policies_response = api.list_namespaced_custom_object(group=CALICO_RESOURCES_GROUP,
+                                                          version=CALICO_RESOURCES_VERSION,
+                                                          namespace=NAMESPACE,
+                                                          plural=CALICO_NETPOL_PLURAL)
+    return policies_response['items']
+
+
+def create_policy(owner, is_sensitive):
+    log_message('Creating policy for [{}{}]'.format(owner, '-sensitive' if is_sensitive else ''))
+    api = get_custom_resource_api()
+    policy_yaml = create_policy_yaml_object(owner, is_sensitive)
+    policy_name_template = policy_yaml[K8S_METADATA_KEY][K8S_OBJ_NAME_KEY]
+    sanitized_owner_name = sanitize_name(owner)
+    policy_name_candidate = policy_name_template.replace(NETPOL_NAME_PREFIX_PLACEHOLDER, sanitized_owner_name)
+    while True:
+        existing_policy_response = api.list_namespaced_custom_object(group=CALICO_RESOURCES_GROUP,
+                                                                     version=CALICO_RESOURCES_VERSION,
+                                                                     namespace=NAMESPACE,
+                                                                     plural=CALICO_NETPOL_PLURAL,
+                                                                     field_selector=K8S_METADATA_NAME_FIELD_SELECTOR
+                                                                     .format(policy_name_candidate))
+        if len(existing_policy_response.get('items')) > 0:
+            log_message('Policy with name [{}] exists already: generating suffix for the current one.'
+                        .format(policy_name_candidate))
+            policy_name_candidate = policy_name_template.replace(NETPOL_NAME_PREFIX_PLACEHOLDER,
+                                                                 sanitized_owner_name + '-' + str(uuid.uuid4())[:8])
+        else:
+            policy_yaml[K8S_METADATA_KEY][K8S_OBJ_NAME_KEY] = policy_name_candidate
+            api.create_namespaced_custom_object(group=CALICO_RESOURCES_GROUP,
+                                                version=CALICO_RESOURCES_VERSION,
+                                                namespace=NAMESPACE,
+                                                plural=CALICO_NETPOL_PLURAL,
+                                                body=policy_yaml)
+            log_message('Policy [{}] created successfully'.format(policy_name_candidate))
+            break
+
+
+def sanitize_name(name: str):
+    return re.sub('[^A-Za-z0-9]+', '-', name).lower()
```

Comment:
Username is often a email, let's split name by `@` sign at first, as it is done in other services. E.g. `mariia_zueva@epam.com` -> `mariia-zueva`